hardware/nic/bcm5700.pan

```diff
@@ -1,14 +1,4 @@
-######################################################
-#
-# RESPONSIBLE: Jacquelin Charbonnel <charbonnel@lal.in2p3.fr>
-#
-######################################################
-
 structure template hardware/nic/bcm5700;
 
-"driver" = "bcm5700";
-"driverrpms" = list("bcm5700", "bcm5700-smp");
-"pxe"    = true;
-"boot"   = false;
-"media"  = "Ethernet";
-"name"   = "Broadcom PCI-X 10/100/1000BASE-T Controller";
+include 'hardware/nic/legacy/bcm5700';
+'type' = 'DEPRECATION WARNING: Use of nic templates immediately under hardware/nic/ is deprecated. Please use templates under hardware/nic/nic_manufacturer/ instead.';
```

Comment:
can you make a structure template `hardware/nic/legacy/deprecated_type`, set this type in there as the only value and include it everywhere (instead of copy/pasting the same line)